modules/node-shared/src/test/scala/io/constellationnetwork/node/shared/infrastructure/block/processing/BlockAcceptanceLogicSuite.scala

```diff
@@ -73,6 +73,15 @@ object BlockAcceptanceLogicSuite extends MutableIOSuite with Checkers {
     }
   }
 
+  test("accept block with signers without collateral when skipped validation") { implicit sc =>
```

Comment:
The most important test is invocation of the GL0 state update function, which has a currency snapshot embedded, with no collateral provided. That's basically the only condition we need to cover, as the E2E can cover the other case.